crypto/include/sodium/crypto_int32.h

```diff
@@ -1,2 +1,3 @@
+#pragma once
```

Comment:
Ideally we would include guard, pragma once can be surprising sometimes (and isn’t *as* portable)

Reply:
?